File: example/android/awn_core/src/main/java/me/carda/awesome_notifications/core/utils/ListUtils.java
package me.carda.awesome_notifications.core.utils;

import java.util.List;

import javax.annotation.Nullable;

public class ListUtils {
    public static boolean isNullOrEmpty(@Nullable List<?> list){
        return list == null || list.isEmpty();
    }

    public static boolean listHasDuplicates(List<Object> list){
        for(int position = 0; position < list.size(); position++)
            for(int searchPointer = position + 1; searchPointer < list.size(); searchPointer++)
                if(CompareUtils.assertEqualObjects(list.get(position), list.get(searchPointer)))
                    return true;
        return false;
    }
}

File: reference/awesome_notifications/ios/AwnCore/Classes/utils/ListUtils.swift
//
//  ListUtils.swift
//  awesome_notifications
//
//  Created by Rafael Setragni on 05/09/20.
//

import Foundation

class ListUtils {

    public static func isNullOrEmpty(_ list: [AnyObject]?) -> Bool {
        return list?.isEmpty ?? true
    }
}
